src/DependencyInjection/Configuration.php

```diff
@@ -112,6 +112,7 @@ private function addContextsSection(ArrayNodeDefinition $node)
                                         ->scalarNode('quality')->defaultValue(80)->end()
                                         ->scalarNode('format')->defaultValue('jpg')->end()
                                         ->scalarNode('constraint')->defaultValue(true)->end()
+                                        ->scalarNode('resizer')->defaultFalse()->end()
```

Comment:
Shouldn‘t website default Null here instead of false @greg0ire ?

Reply:
Uuuuh yeah probably, indeed!